common/services/whats-new-content.js

```diff
@@ -49,22 +49,30 @@ const service = {
       return null
     }
 
+    for (let i = 0; i < entries.items.length; i++) {
+      const currentItem = entries.items[i]
+      currentItem.fields.body = service.convertToHTMLFormat(
+        currentItem.fields.body
+      )
+      currentItem.fields.date = service.formatDate(currentItem.fields.date)
+    }
```

Comment:
```suggestion
    const entries = entries.items.forEach(({ fields: { title, body, date } }) => (
      {
        title: title,
        body: service.convertToHTMLFormat(body),
        date: service.formatDate(date)
      }
    ))
```

I wonder if something like this is slightly nicer, allows us to be more explicit on which fields we want to pass through to the template as well?

Reply:
I do like this, nicer to read 🚀